app/main.ipynb

```diff
@@ -237,18 +305,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 23,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "np.sum(A, axis=0)",
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T13:20:47.632326Z",
-     "start_time": "2023-07-03T13:20:47.621552Z"
+     "end_time": "2024-11-07T10:09:20.162893Z",
+     "start_time": "2024-11-07T10:09:20.157818Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([10, 42])"
+      ]
+     },
```

Comment:
are you sure it's a correct answer? you have 5 values by Y axis but only 2 values are returned